Flagsmith.FlagsmithClient/Cache/IdentityFlagListCache.cs

```diff
@@ -1,13 +1,21 @@
-﻿using Flagsmith.Providers;
+﻿using System;
+using System.Collections.Generic;
+using Flagsmith.Providers;
 
 namespace Flagsmith.Cache
 {
     internal class IdentityFlagListCache : FlagListCache
     {
         private readonly IdentityWrapper _identityWrapper;
 
-        public IdentityFlagListCache(IdentityWrapper identityWrapper, IFlags flags, IDateTimeProvider dateTimeProvider, int cacheDurationInMinutes) :
-            base(dateTimeProvider, flags, cacheDurationInMinutes)
+        public IdentityFlagListCache(IdentityWrapper identityWrapper,
+            IDateTimeProvider dateTimeProvider,
+            AnalyticsProcessor analyticsProcessor,
+            Func<string, IFlag> defaultFlagHandler,
+            int cacheDurationInMinutes) :
+            base(dateTimeProvider,
+                new Flags(new List<IFlag>(), analyticsProcessor, defaultFlagHandler),
```

Comment:
Can't we just make `flags` nullable in the base class and scrap all of this?

Reply:
Yes I think it could work! Give me a minute!

Reply:
fixed